scripts/get_gnomad_lof_variants.py

```diff
@@ -134,30 +185,41 @@ def add(a, b):
     return hl.or_else(a, 0) + hl.or_else(b, 0)
 
 
+SUPPORTED_GNOMAD_VERSIONS = (2, 4)
+
+
 def get_gnomad_lof_variants(gnomad_version, gene_ids, include_low_confidence=False):
-    if gnomad_version not in (2, 3):
+    if gnomad_version not in SUPPORTED_GNOMAD_VERSIONS:
         raise Exception(f"Invalid gnomAD version {gnomad_version}")
 
     if gnomad_version == 2:
         ds = load_gnomad_v2_variants()
-    elif gnomad_version == 3:
-        ds = load_gnomad_v3_variants()
+    elif gnomad_version == 4:
+        ds = load_gnomad_v4_variants()
 
     reference_genome = "GRCh37" if gnomad_version == 2 else "GRCh38"
     genes = [fetch_gene(gene_id, reference_genome) for gene_id in gene_ids]
 
+    chrom_prefix = "chr" if reference_genome == "GRCh38" else ""
+
     ds = hl.filter_intervals(
         ds,
         [
             hl.parse_locus_interval(
-                f"{gene['chrom']}:{gene['start']}-{gene['stop']}", reference_genome=reference_genome
+                f"{chrom_prefix}{gene['chrom']}:{gene['start']}-{gene['stop']}",
+                reference_genome=reference_genome,
             )
             for gene in genes
         ],
     )
 
     gene_ids = hl.set(gene_ids)
 
+    if gnomad_version == 2:
```

Comment:
Let's cut this branch and the `annotate_gnomad_v2_etc` function it calls until such time, which is as you note above possibly never, that we actually need it.